Glyssen/ProjectExporter.cs

```diff
@@ -310,6 +337,53 @@ internal DataTable GeneratePreviewTable()
 			}
 		}
 
+		private void GenerateClipFiles(string directoryPath)
+		{
+			const int kLineNumIndex = 0;
+			const int kBookIndex = 2;
```

Comment:
We should use the ExportColumn enum instead of these constants. These will be off by one if the Actor column is present. I'm checking with Duane, but I think the option of generating clip files should only be available if there are actor assignments.